libraries/botbuilder-dialogs-adaptive/src/languageGeneratorExtensions.ts

```diff
@@ -36,6 +36,11 @@ export const languagePolicyKey = Symbol('LanguagePolicy');
  * Extension methods for language generator.
  */
 export class LanguageGeneratorExtensions {
+    private static readonly _languageGeneratorManagers: Map<ResourceExplorer, LanguageGeneratorManager> = new Map<
```

Comment:
Generally, the type can be inferred by initialization. In this case you can omit just write `private static readonly _languageGeneratorManagers = new Map<ResourceExplorer, LanguageGeneratorManager>();`